src/app/profile/my-spaces/my-spaces-search-dialog/my-spaces-search-spaces-dialog.component.ts

```diff
@@ -8,10 +8,11 @@ import {
 
 import {
   BehaviorSubject,
-  Observable,
   Subject,
   Subscription
 } from 'rxjs';
+import { combineLatest } from 'rxjs/observable/combineLatest';
```

Comment:
Please convert:

```
import { combineLatest } from 'rxjs/observable/combineLatest';
a$.pipe(combineLatest(b$, c$));
```

into:

```
import { combineLatest } from 'rxjs';
combineLatest(a$, b$, c$);
```

Reply:
`combineLatest` is not in `rxjs` for version 5.5.2 I think.